--- .xinitrc	2026-08-07 19:43:37
+++ .xinitrc2	2026-08-07 19:43:53
@@ -5,14 +5,14 @@
 export WM="/usr/bin/wmaker"
 
 #xrandr --auto
-#xrandr --newmode "2560x1440_60.00" 312.25 2560 2752 3024 3488 1440 1443 1448 1493 -hsync +vsync
-#xrandr --addmode HDMI-2 "2560x1440_60.00"
-#xrandr --output HDMI-2 --mode "2560x1440" --right-of eDP-1
+xrandr --newmode "2560x1440_60.00" 312.25 2560 2752 3024 3488 1440 1443 1448 1493 -hsync +vsync
+xrandr --addmode HDMI-2 "2560x1440_60.00"
+xrandr --output HDMI-2 --mode "2560x1440" --right-of eDP-1
 
 xrdb -merge ~/.Xresources
 xmodmap ~/.Xmodmap
-#xset m 6 4
-xset m 1.5 2
+xset m 6 4
+#xset m 1.5 2
 xset r rate 192 32
 
 /usr/bin/uim-xim &
