*** .xinitrc	2026-10-03 17:51:31.792835801 +0900
--- .xinitrc2	2026-10-03 17:51:48.379872988 +0900
*************** export QT_IM_MODULE=uim
*** 5,13 ****
  export WM="/usr/bin/wmaker"
  
  #xrandr --auto
! #xrandr --newmode "2560x1440_60.00" 312.25 2560 2752 3024 3488 1440 1443 1448 1493 -hsync +vsync
! #xrandr --addmode HDMI-2 "2560x1440_60.00"
! #xrandr --output HDMI-2 --mode "2560x1440" --right-of eDP-1
  
  xrdb -merge ~/.Xresources
  xmodmap ~/.Xmodmap
--- 5,13 ----
  export WM="/usr/bin/wmaker"
  
  #xrandr --auto
! xrandr --newmode "2560x1440_60.00" 312.25 2560 2752 3024 3488 1440 1443 1448 1493 -hsync +vsync
! xrandr --addmode HDMI-2 "2560x1440_60.00"
! xrandr --output HDMI-2 --mode "2560x1440" --right-of eDP-1
  
  xrdb -merge ~/.Xresources
  xmodmap ~/.Xmodmap
